src/Grace/Repl.hs

```diff
@@ -95,8 +113,14 @@ repl = do
 
                     liftIO (Pretty.renderIO True width IO.stdout (Pretty.pretty type_ <> "\n"))
 
+    let quit _ =
+            liftIO (throwIO Interrupt)
+
     let options =
-            [ ("let", Repline.dontCrash . assignment)
+            [ ("help", Repline.dontCrash . help)
+            , ("let", Repline.dontCrash . assignment)
+            , ("paste", Repline.dontCrash . \_ -> return ())
```

Comment:
You know I can't help myself 🙂:

```suggestion
            , ("paste", Repline.dontCrash . mempty)
```

Also, maybe add a comment explaining that this is added so that it's suggested by auto-completion